script.module.resolveurl/lib/resolveurl/plugins/govad.py

```diff
@@ -22,8 +22,8 @@
 
 class GoVadResolver(ResolveGeneric):
     name = 'GoVad'
-    domains = ['govad.xyz']
-    pattern = r'//(.*\.?govad\.xyz)/(?:embed-)?([0-9a-zA-Z-]+)'
+    domains = ['govad.xyz', 'goveed.autos']
+    pattern = r'//(.*\.?(?:govad|goveed)\.(?:xyz|autos))/(?:embed-)?([0-9a-zA-Z-]+)'
```

Comment:
Can be done slightly better
`    pattern = r'//(.*\.?gov[ae]*d\.(?:xyz|autos))/(?:embed-)?([0-9a-zA-Z-]+)'`